docs/tutorial/index.rst

```diff
@@ -26,3 +26,4 @@ code into container applications:
    4. Bundle a Node.js app within a rock <node-app.rst>
    5. Build a rock for a Flask application <flask.rst>
    6. Build a rock for a Django application <django.rst>
+   6. Build a rock for a Go application <go.rst>
```

Comment:
Perhaps make it `7.`? I think FastAPI has 7. as well, I'm guessing whichever is merged first, the other will get a merge conflict